docs/tutorials-samples/vs-golden-path.md

```diff
@@ -0,0 +1,107 @@
+Building simple solutions with Visual Studio
+============================================
+
+The scripts in this document describe the steps necessary to build a number of typical .NET Core solutions, or solutions that include .NET Core components, using Visual Studio. The scenarios include testing, and using third-party libraries that have not been explicitly built for the most recent version of .NET Core.
+
+Prerequisites
+-------------
+
+These scripts assume that .NET Core CLI (latest RC2 version), Visual Studio 2015 Update 2, and Visual Studio tooling (latest RC2 version) are installed on the machine.
+
+A solution using only .NET Core projects
+----------------------------------------
+
+### Writing the library
+
+1. File -> New. Under Templates / .NET Core, choose "Class Library (.NET Core)" and name the project "Library" and the solution "Golden".
+2. Under References, manage NuGet packages, use "nuget.org" as the package source, and browse (including prerelease) for `Newtonsoft.Json`. Install.
+3. Go back to packages and install `Microsoft.NETCore.Portable.Compatibility` from the Core MyGet package source (use the latest rc2 version). Restore packages. (\*)
+4. Open `project.json` and under frameworks/netstandard1.5, add `"imports": ["portable-net40+sl5+win8+wp8+wpa81", "dotnet"]`. Restore packages. (\*)
+5. Rename the class "Thing". Add a method: `public int Get(int number) => Newtonsoft.Json.JsonConvert.DeserializeObject<int>($"{number}");`
+6. Verify that the solution builds.
+
+### Writing the test project
+
+1. Create a "test" solution folder. This is only a solution folder, not a physical folder.
+2. Right-click the new "test" solution folder, add/new .NET Core class library project, call it "TestLibrary" and explicitly put it under `Golden/test`.
+3. Right-click References and choose "Add reference". Check "Library" under Projects/solution.
+4. Add references to `Newtonsoft.Json`, `Microsoft.NETCore.Portable.Compatibility`, and add `"imports": [ "portable-net40+sl5+win8+wp8+wpa81", "dotnet" ]` to `frameworks/netstandard1.5` in `project.json`. Restore packages. (\*)
+5. Add the `xUnit` and `xunit.runner.visualstudio` packages from nuget.org to the project. Use "portable-net45+win8+wp8+wpa81" in the imports in project.json.
+6. Go back to packages and install `Microsoft.NETCore.Portable.Compatibility` from the Core MyGet package source (use the latest rc2 version). Restore packages. (\*)
+7. Rename `Class1.cs` to `LibraryTests.cs`, add the following code to the class:
+    ```csharp
+        [Fact]
+        public void ThingGetsObjectValFromNumber() {
+            Assert.Equal(42, new Thing().Get(42));
+        }
+    ```
+8. Build. Test -> Windows -> Test Explorer. Run all tests. (this doesn't work)
```

Comment:
I see you say this doesn't work.  Are these the steps you are expecting to eventually work or are you expecting some flavor of this story to work right now?